helm/circles-infra-suite/values-staging.yaml

```diff
@@ -47,7 +47,8 @@ contracts:
   proxyFactoryAddressCRC: 0x0F9f9C38E53674EF7a29A1DE38EAbF34e1DCB81C
   safeAddressCRC: 0x8b4404DE0CaECE4b966a9959f134f0eFDa636156
   proxyFactoryAddress: 0x0F9f9C38E53674EF7a29A1DE38EAbF34e1DCB81C
-  safeAddress: 0x8b4404DE0CaECE4b966a9959f134f0eFDa636156 
+  safeAddress130: 0x8b4404DE0CaECE4b966a9959f134f0eFDa636156 
+  safeAddress: 0x8b4404DE0CaECE4b966a9959f134f0eFDa636156
```

Comment:
For the staging values we actually don't know the addresses yet

Reply:
the values dont make sense in staging but that we know, bc we dont have those contracts in sokol